src/pages/en/index.astro

```diff
@@ -0,0 +1,9 @@
+---
+import HeroSection from "../../components/HeroSection.astro";
+import Layout from "../../layouts/Layout.astro";
+
+---
+
+<Layout>
+    <HeroSection locale="en" />
```

Comment:
No i tutaj do kazdej sekcji bedzie trzeba przypisać locale. Dzieki temu przy buildzie produkcyjnym bedziemy mieli od razu juz dwa indexy html dla dwoch roznych jezykow.

Reply:
Chyba ze masz lepszy pomysl na to. To jest taki najprostszy i najszybszy

Reply:
ogolnie wyglada spoko mozemy isc w takie rozwiazanie. jedynie kwestia tego, ze wlasnie trzeba locale przekazywac za kazdym razem. 
rownie dobrze, zamiast wywolywac `getTranslation` za kazdym razem w komponentach, mozemy przekazywac do komponentow okreslony content zamiast samego `locale` - ktore nam niewiele wnosi poza byciem 'flaga'

Reply:
no i ogolnie powinnismy uzywac markdown zamiast json do contentu. wiem, ze robiles json bo chodzilo o sam mechanizm tlumaczen. ale tak zebysmy pamietali ze bedziemy musieli operowac na innym formacie danych bo beda w contentcie boldy, linki etc.

Reply:
ogolnie astro ma w docsach cos takiego jak content collections 
https://docs.astro.build/en/guides/content-collections/
i pozniej cala taka kolekcje mozna nawet filtrowac przy podpinaniu sie po nia: 
<img width="489" alt="{CCE7A599-D529-4458-9D69-AC4910658F59}" src="https://github.com/user-attachments/assets/794e04a6-1416-4b68-a9bf-f7e49279e4cd" />

ale wiem, ze to wiecej zachodu ;s bo tam pozniej tez mozna otypowac w Zodzie i inne blablabla.. pewnie tego wszystkiego nie potrzebujemy.

Reply:
super to wymysliles! nie mam zadnych uwag! 
![Uploading GoodJobCryptoGIF.gif…]()